### DISCUSSION TITLE
Navigation Limit on GitHub Web Interface

### DISCUSSION THREAD

**Discussion by moizahmedtuc** [community/community#134905] [Other Feature Feedback, Questions, & Ideas]
ISSUE ⚠️ Navigation Limit on GitHub Web Interface

Description:

The GitHub web interface initially shows **123k** files for the below query, it does not allow navigating beyond the 5th page of results.

Query Used:

- "new(params[" OR "create(params[" OR "update_attributes(params[" path:app/controllers language:Ruby
- URL: [GitHub Query Search URL](https://github.com/search?q=%22new%28params%5B%22+OR+%22create%28params%5B%22+OR+%22update_attributes%28params%5B%22+path%3Aapp%2Fcontrollers+language%3ARuby&type=code&p=1)

Question❓ Is there a limitation on how many pages or results one can navigate through on the GitHub website? If so, then why?

**Comment by LiteBrite82** [comment]
Hi @moizahmedtuc 

Thanks for posting! 

[Only 100 results? #9868](https://github.com/orgs/community/discussions/9868) is an already existing discussion on this topic. Please continue the community conversation there so we can track the topic. 

Thanks again!
